crates/tracing-instrumentation/src/prometheus_metrics.rs

```diff
@@ -74,7 +81,23 @@ impl Prometheus {
     }
 
     pub fn global_labels(&self) -> &Vec<String> {
```

Comment:
Have you tested that the node id shows up for all metrics? To me it looks as if adding the node id to those labels, will only make it show up for the rocksdb metrics. I think it would have to go to `PrometheusBuilder::add_global_label()`.

Reply:
If we have the node_id do we still want to keep the node_name? Being mindful about the data size that needs to be scraped.

Reply:
I did not test this very well! Indeed, this only adds the node_id to the rocksdb cf metrics.

If we agree to only use node_id, let's take out the node_name label at the same time?